modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/string.ts

```diff
@@ -12,6 +12,30 @@
  * details.
  */
 
+const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+
+/**
+ * Get the label according to the locale
+ */
+
+export function getLocalizableLabel(
+	labels: LocalizedValue<string> | undefined,
+	creationLanguageId: Locale,
```

Comment:
@alinedoleron could you sort the parameters alphabetically, please? Thanks :)

Reply:
Thanks, @naraandrade. I could only sort the non optional parameters. The build complains if a sort all the parameters: 

> "A required parameter cannot follow an optional parameter."

Reply:
Ok, Thank you!